Automattic/wp-calypso
# composite-checkout: Move wpcom dir to wpcom package

#### Changes proposed in this Pull Request

While developing the `composite-checkout` package, we started creating host-specific components in the `src/wpcom` directory. These components should be moved into the `composite-checkout-wpcom` package instead as they are wpcom-specific and will not be used by other hosts. This PR aims to do that.

#### Testing instructions

- Add a file called `packages/composite-checkout/demo/private.js` and inside it put the following: `export const stripeKey = 'TEST KEY HERE'`;
- Run `npm run composite-checkout-demo`.
- Verify that the form works as expected.
- Run calypso with the `composite-checkout-wpcom` flag and verify that the form works as expected there as well.
